lab_3_ann_retriever/main.py

```diff
@@ -49,6 +52,18 @@ def calculate_distance(query_vector: Vector, document_vector: Vector) -> float |
 
     In case of corrupt input arguments, None is returned.
     """
+    if not (isinstance(query_vector, tuple) and isinstance(document_vector, tuple)
+            and all(isinstance(cor, float) for cor in query_vector)
+            and all(isinstance(cor, float) for cor in document_vector)
+            and (len(query_vector) == len(document_vector) or len(query_vector) == 0 or len(
+                document_vector) == 0)):
+        return None
+    if not query_vector or not document_vector:
+        return 0.0
```

Comment:
Strange. Should be `None`

Reply:
fixed

Reply:
However, test_calculate_distance_for_empty_vectors is failing now. I thought, if one of vectors is empty, than distance should be 0?

Reply:
So, i left 0.0